### ISSUE TITLE
replace integers on test definitions with typestrings?

### ISSUE THREAD

**Issue by thinlizzy** [nemomobile/tut#2]
Have you checked this? https://github.com/irrequietus/typestring

This is a simple header file that allows strings to be defined as types. Tests could be defined like:

``` (cpp)
template<>
template<>
void object::test< typestring_is("happy") >() { ... }
```

This could replace the annoying integers on test definitions and also it gives a meaningful context to the test member function.

**Comment by deztructor** [comment]
@thinlizzy yes, I saw it, thanks :) it looks appealing. I wonder do tut upstream develolers know about it
